jbpm-db-scripts/src/main/resources/db/upgrade-scripts/oracle/jbpm-7.58-to-7.59.sql

```diff
@@ -0,0 +1,5 @@
+create unique index IDX_CorrelationKeyInfo_name on CorrelationKeyInfo(name);
```

Comment:
This is already part of the springboot schema
https://github.com/kiegroup/jbpm/blob/940bcf5698db5d95a911291f2c2cf4b1930fac4a/jbpm-db-scripts/src/main/resources/db/ddl-scripts/oracle/oracle-springboot-jbpm-schema.sql#L101

Are we sharing the same upgrade script for both?

Reply:
I was suprised by that, but only happens with the autogenerated means (bytea and new spring boot scripts). We cannot do much about it but add some release notes